src/modules/commentsApi.js

```diff
@@ -0,0 +1,58 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const appId = '8ge52RZ4xe8gKMYNyxDN';
+
+// Function to create a new app
+/* async function createApp() {
+  const url = `${baseUrl}apps/`;
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+    });
+    if (response.ok) {
+      const data = await response.text(); // Get the response as text
+      appId = data.trim(); // Trim whitespace from the response
+      console.log('App created. App ID:', appId);
+    } else {
+      console.error('Failed to create app:', response.status, response.statusText);
+    }
+  } catch (error) {
+    console.error('Error creating app:', error);
+  }
+} */
+
+// Function to create a new comment for an item
+export async function createComment(itemId, username, comment) {
+  const url = `${baseUrl}apps/${appId}/comments`;
+  const data = {
+    item_id: itemId,
+    username,
+    comment,
+  };
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    return response.status === 201;
+  } catch (error) {
+    return false;
+  }
+}
+
+// Function to get comments for a specific item
+export async function getItemComments(itemId) {
+  const url = `${baseUrl}apps/${appId}/comments?item_id=${itemId}`;
+  try {
```

Comment:
- [ ] Great job on successfully implementing the project requirements! 👏 Your codebase is clean and well-organized, which is commendable. However, as a suggestion for improvement, I recommend using modern JavaScript `ES6` syntax throughout your codebase.

    - One way to achieve this is by utilizing arrow functions (`=>`) instead of the traditional `function` keyword when declaring your functions. Arrow functions offer a more concise and expressive way of defining functions, which can enhance the readability of your code.

     - By adopting `ES6` syntax, your code will be more in line with current industry standards, making it easier for other developers to understand and maintain your codebase.

Overall, your work is impressive. Happy coding! 😊🚀